requirements.txt

```diff
@@ -1,3 +1,3 @@
 beautifulsoup4>=4.4, <5.0
 requests>=2.5.3, <3.0
-slacker==0.9.30
+lxml==3.7.3
```

Comment:
Might want to note in the README that `libxml` is now a system library dependency.